9-regular-expressions/09-regexp-quantifiers/article.md

```diff
@@ -66,77 +66,78 @@ There are shorthands for most used quantifiers:
     ```
 
 `pattern:?`
-: Means "zero or one", the same as `pattern:{0,1}`. In other words, it makes the symbol optional.
+: Representa "zero ou um da captura anterior", equivalente ao `pattern:{0,1}`; marca partes da expressão como opcionais.
 
-    For instance, the pattern `pattern:ou?r` looks for `match:o` followed by zero or one `match:u`, and then `match:r`.
+    O padrão `pattern:ou?r`, por exemplo, busca por `match:o` seguido de zero ou um `match:u`, seguido de um `match:r`.
 
-    So, `pattern:colou?r` finds both `match:color` and `match:colour`:
+    Dessa forma, `pattern:colou?r` reconhece ambos `match:color` e `match:colour`:
 
     ```js run
+    // Devo escrever cor (em inglês americano) ou cor (em inglês britânico)?
```

Comment:
Is this commentary necessary?

I think that all English fluent people should know the difference between the US and UK English, the same way all Portuguese fluent people should know the difference between BR and Non-BR Portuguese.

Reply:
The comment is here as an explanation for the reader, since the example used by Ilya only makes sense in English, and I couldn't think of a good enough example in Portuguese.

To explain a bit further, this examples highlights the use of the `?` quantifier by showcasing an expression which matches both the North American spelling "color" and the British spelling "colour", which has an added `u`.

To better explain the concept, I kept the original phrase and translated its meaning above it, but I think I can make this distinction a bit clearer!